docker/1.6.0/py2/Dockerfile.gpu

```diff
@@ -42,6 +47,67 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && ln -s $(which python) /usr/local/bin/python
 
+# Install cuda-dev libraries as a dependency for Horovod with MXNet backend
+ENV CUDA_LIB_URL=https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64
+RUN wget ${CUDA_LIB_URL}/libcublas10_10.2.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/libcublas-dev_10.2.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb \
+ && dpkg -i libcublas10_10.2.1.243-1_amd64.deb \
+    libcublas-dev_10.2.1.243-1_amd64.deb \
+    cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb \
+ && apt-get install -f \
+ && rm libcublas10_10.2.1.243-1_amd64.deb \
+    libcublas-dev_10.2.1.243-1_amd64.deb \
+    cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb
+
+###########################################################################
+# Horovod dependencies
+###########################################################################
+
+# Install Open MPI
+RUN mkdir /tmp/openmpi \
+ && cd /tmp/openmpi \
+ && curl -fSsL -O https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.1.tar.gz \
+ && tar zxf openmpi-4.0.1.tar.gz \
+ && cd openmpi-4.0.1 \
+ && ./configure --enable-orterun-prefix-by-default \
+ && make -j $(nproc) all \
+ && make install \
+ && ldconfig \
+ && rm -rf /tmp/openmpi
+
+RUN apt-get update \
+ && apt-get install -y \
+    python \
+    python-pip
```

Comment:
Why do we need these lines? It looks like we've already done this before

Reply:
Good catch.
+1

Reply:
Done